src/connection/mod.rs

```diff
@@ -6,6 +6,7 @@ use tokio::net::TcpStream;
 mod connection;
 mod tracker;
 
+pub use crate::server::Context;
```

Comment:
Should this be `pub`? Did you mean to re-export a struct defined publicly elsewhere in your crate?

Reply:
Whoops! I think I did this when I wasn't sure where I wanted it to live, and forgot to remove this. Good catch. 90b110a